### PR TITLE
Update Meterpreter file existence tests for CI environments

### PR DESCRIPTION
**Pull request by AlanFoster**
Update Meterpreter file existence tests for CI environments

### CODE REVIEW

File: test/modules/post/test/file.rb [hunk 1]
```diff
@@ -128,7 +128,8 @@ def test_file
         '/etc/passwd',
         '/etc/master.passwd',
         '%WINDIR%\\system32\\notepad.exe',
-        '%WINDIR%\\system32\\calc.exe'
+        '%WINDIR%\\system32\\calc.exe',
+        File.expand_path(__FILE__)
```
Comment (AlanFoster) [inline test/modules/post/test/file.rb:132]: calc.exe, notepad.exe etc won't exist on most Windows CI boxes. There's probably a file that will _always_ exist, but pointing to the current test module works when running tests directly on the windows machine.

Comment (jmartin-r7) [reply test/modules/post/test/file.rb:132]: Consider maybe `%WINDIR%\\system32\\drivers\\etc\\hosts` I suspect that would rarely be removed.

Comment (OJ) [reply test/modules/post/test/file.rb:132]: the `.ini` files in `%WINDIR%` are pretty reliable I think. eg. `system.ini`, `win.ini`

Comment (AlanFoster) [reply test/modules/post/test/file.rb:132]: Thanks! Looks like `%WINDIR%\system.ini` and `%WINDIR%\system.ini` exist on the CI box, as does the host file. Interestingly the hostfile for Github's CI boxes has a bunch of crypto hostname entries resolving to 127.0.0.1

That aside - it turns out the real issue is the expansion of `%WINDIR%` only happens in the windows Meterpreter.

Some example paths ran across the Meterpreters:

||python3|Java|Windows
|------------------|---------|--------|-|
| c:\boot.ini | ✗ | ✗ | ✗|
| c:\pagefile.sys | ✗ | ✗ | ✗|
| /etc/passwd | ✗ | ✗ | ✗|
| /etc/master.passwd | ✗ | ✗ | ✗|
| c:\windows\system32\drivers\etc\hosts | ✓ | ✓ | ✓|
| %WINDIR%\system32\drivers\etc\hosts | ✗ | ✗ | ✓|
| c:\Windows\win.ini | ✓ | ✓ | ✓|
| %WINDIR%\win.ini | ✗ | ✗ | ✓|
| C:\Windows\notepad.exe | ✓ | ✓ | ✓|
| C:\Windows\calc.exe | ✗ | ✗ | ✗|
| c:\Windows\notepad.exe | ✓ | ✓ | ✓|
| %WINDIR%\notepad.exe | ✗ | ✗ | ✓|
| %WINDIR%\calc.exe | ✗ | ✗ | ✗|
| %WINDIR%\system32\notepad.exe | ✗ | ✗ | ✓|
| %WINDIR%\system32\calc.exe | ✗ | ✗ | ✓|
| D:/a/metasploit-framework/metasploit-framework/test/modules/post/test/file.rb | ✓ | ✓ | ✓ |

I'll create a separate test/issue for deciding if this is behavior we want to align across the Meterpreters.
Note there's no data available for PHP + Windows as the test crashes before it reaches this point

**Pull request merged**